src/node/node.ts

```diff
@@ -36,28 +37,43 @@ export class Node implements ServerConfig {
     nodeReq: IncomingMessage
   ): Promise<HttpRequest> {
     return {
-      body: await streamToString(nodeReq),
+      body: new HttpBodyImpl(toReadableStream(nodeReq)),
```

Comment:
You can define this as another static function of HttpBodyImpl. Something like fromNodeStream.